dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype.properties

```
@@ -122,7 +122,7 @@ xPoweredBy.name=X-Powered-By
 xPoweredBy.desc=Alternative string for the X-Powered-By header setting. The equivalent custom property in the full application server profile is com.ibm.ws.webcontainer.xpoweredby. There is no default value for this property. If the property is not set, the value of the X-Powered-By header is set to Servlet/<servlet spec version>, as defined by the Servlet specification.
 
 disableXPoweredBy.name=Disable the X-Powered-By header
-disableXPoweredBy.desc=Disable setting the X-Powered-By header. The equivalent custom property in the full application server profile is com.ibm.ws.webcontainer.disablexpoweredby. For servlet-5.0 and newer Servlet features, the default value is true.
+disableXPoweredBy.desc=Disable setting the X-Powered-By header. The equivalent custom property in the full application server profile is com.ibm.ws.webcontainer.disablexpoweredby. For servlet-5.0 and newer Servlet features, the default value is true. Only configurable for Servlet 5.0 and older Servlet features.
```

This is a little confusing, in part because the description doesn't tell you anything about the significance of setting the header, it just tells you what the property does, not why you might need to set it. The default per version is a little confusing too. I _think_ the following applies:
- in servlet-5.0 and later, by default no X-Powered-By header is set.
- in servlet 4.0 and earlier, the X-Powered-By header is set by default
- in servlet 5.0 and earlier, you can use this property to configure whether the X-Powered-By header is set. But in subsequent versions, you will not have the option to set an X-Powered-BY header.

if that's correct, then maybe something like:
```suggestion
disableXPoweredBy.desc=When this property is set to true,  the X-Powered-By header is not set. In servlet-5.0 and later, the default value is true. In servlet-4.0 and earlier, the X-Powered-By header is set by default. This property is configurable only in servlet-5.0 and earlier. The equivalent custom property in the full application server profile is com.ibm.ws.webcontainer.disablexpoweredby.
```

---

I've made the suggested changes and started a new personal build.